packages/flutter/lib/src/scheduler/scheduler.dart

```diff
@@ -0,0 +1,149 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:collection/priority_queue.dart';
+import 'package:flutter/animation.dart' as animation show scheduler;
+
+typedef Task();
+
+/// An entry in the scheduler's priority queue.
+///
+/// Combines the task and its priority.
+class _SchedulerEntry {
+  final Task task;
+  final int priority;
+
+  _SchedulerEntry(this.task, this.priority);
+}
+
+class Priority {
+  static const kIdle = const Priority._(0);
+  static const kAnimation = const Priority._(100000);
+  static const kTouch = const Priority._(200000);
+
+  /// Relative priorities are clamped by this offset.
+  ///
+  /// It is still possible to have priorities that are offset by more than this
+  /// amount by repeatedly taking relative offsets, but that's generally
+  /// discouraged.
+  static const kMaxOffset = 10000;
+
+  final int _value;
+
+  int get value => _value;
```

Comment:
put the getter just above the field, so that when we /// dartdoc the getter, the comment is in a block just above the getter and the field (otherwise the dartdocs comments end up splitting the field and the getter)